model/ddl.go

```diff
@@ -429,4 +429,14 @@ type SchemaDiff struct {
 	OldTableID int64 `json:"old_table_id"`
 	// OldSchemaID is the schema ID before rename table, only used by rename table DDL.
 	OldSchemaID int64 `json:"old_schema_id"`
+
+	OtherAffectedSchema []*SchemaAffect `json:"other_affected_schema"`
```

Comment:
```suggestion
	OtherAffectedSchemas []*SchemaAffect `json:"other_affected_schema"`
```

Reply:
I don't think the "SchemaAffect" using "Schema" as a unit is particularly good. "Schema" in "OtherAffectedSchema" and "SchemaID" is a bit difficult to understand.